core/includes/bootstrap.inc

```diff
@@ -3545,17 +3571,17 @@ function language_multilingual() {
 /**
  * Returns a list of configured languages.
  *
- * @param $only_enabled
+ * @param bool $only_enabled
  *   (optional) Whether to return only enabled languages.
- * @param $option_list
+ * @param bool $option_list
  *   (optional) Whether to return a list of options instead of objects. This
  *   list is suitable for use in a select list #options array. This list is
  *   ordered by weight and then language name.
  * @param bool $native_names
  *   (optional) Whether the option_list labels should use the native name
  *   instead of the English language name. Only applies if $option_list is TRUE.
  *
- * @return (stdClass|string)[]
+ * @return array
```

Comment:
I think this can be instead: `(object|string)[]`.

I found this reference: https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html